### PR TITLE
Insulated gloves crate requires engineering access

### PR DESCRIPTION
**Pull request by brndd**
The station-wide greyshirt all access must end. Crate access requirements don't deter anyone anyway.

:cl:
 * rscdel: the electrical maintenance crate, which contains insulated gloves, now requires engineering access to open.

### CODE REVIEW

**Comment by Kharkov72** [comment]
_The greyshit metaclub will remember this._

**Comment by bathosbathtime** [comment]
Malf buff

**Comment by Matidash** [comment]
Hacking doors open doesn't require gloves though. Clearly we need to ID lock multitools and signalers 
At best this is just a buff to rogue AIs and a pain for people doing autism projects as non engineers

**Comment by ShiftyRail** [comment]
There is absolutely no need to get insulated glubbs to hack doors safely, it just takes longer to test the wires.
It does, however, fuck everyone who need to deal with stuff that is permanently electrified. As a few examples of situations that are made harder to deal with :

- malf/rogue borgs (just shock my everything up)
- anything involving replacement/removal of light fixtures (ghetto bar autism)
- anything involving APC hacking in case there is no power on the station

In short, it doesn't "punish" the people it want to punish but fuck over everyone else. There is besides very little reason for this PR from the start

**Comment by soyconsumer** [comment]
This PR triggers powergay black crayon sec more than assistants

**Comment by Kammerjunk** [comment]
basically what Shifty said
if I don't get glubbs, I've always just hacked the doors at roundstart because the shock isn't enough to kill you
edit: also whew

**Comment by brndd** [comment]
Let it be known that I'm the shitcurity who confiscates glubbs from greyshirts the moment they step out of line. This change has been a long time coming.

**Comment by jknpj** [comment]
:thinking:

**Comment by FudgePucker** [comment]
I want an admin to say that it is allowed for me to throw gray tiders into the engine if they smash their way into engineering for gloves before I can even remotely support this PR.

Fuck off, we only have six pairs.

**Comment by Kammerjunk** [comment]
well confiscate their glubbs the moment they step out of line
that's an interaction

**Comment by Lokiamis** [comment]
Everything in the crate can be found in public rooms on the station. Gloves are not illegal. There's no reason to lock it. Spitecoding.

**Comment by ghost** [comment]
Joke PR.

**Comment by SonixApache** [comment]
ayy lmao

**Comment by ghost** [comment]
You've doomed us all, Sonix! How _could_ you?!

**Comment by Durgen** [comment]
>making a pull request that would be impossible to merge

what did he mean by this

**Comment by Exxion** [comment]
Sure looks from the above screenshot like this wasn't even intended to go through
